metadata/calendar.json

```diff
@@ -10,6 +10,21 @@
                 "name": "CalendarClient",
                 "displayName": "Google Calendar Client",
                 "operations": [
+                    {
+                        "name": "getCalendars",
+                        "displayName": "Get calendars",
+                        "parameters" : [
+                            {
+                                "name": "optional",
+                                "displayName": "Optional query parameters"
+                            }
+                        ],
+                        "returnValue": {
+                            "name": "stream<Calendar>",
+                            "displayName": "Stream of Calendars"
+                        }
+        
```

Comment:
Shall we remove this new line?

In the calendar.json, same issue is there in some other places also. Shall we fix them as well?